diagnostic.md

```diff
@@ -92,7 +123,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+text color will be red, font family will be arial, since the two class selectors resolve in the order they appear in the css file.
```

Comment:
The text color will be yellow from ```.specialDiv``` and the font-family will be ```cursive``` from the inline style.